.github/template/version-branch-template.yml

```diff
@@ -34,6 +34,9 @@
 '15.2':
   engine_branch: BABEL_3_1_STABLE__PG_15_2
   extension_branch: BABEL_3_1_STABLE
+'15.3':
```

Comment:
Need to update 14.8 entry above with the stable branches and add a new entry for 14.9 with dev branches.